test/operators/update-fee.ts

```diff
@@ -22,36 +22,41 @@ describe('Operator Fee Tests', () => {
     )).to.emit(ssvNetworkContract, 'OperatorFeeDeclared');
   });
 
+  it('Declare fee with zero value emits "OperatorFeeDeclared"', async () => {
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[2]).declareOperatorFee(1, 0
+    )).to.emit(ssvNetworkContract, 'OperatorFeeDeclared');
+  });
+
   it('Declare a lower fee gas limits', async () => {
-    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[2]).declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
   });
 
   it('Declare a higher fee gas limit', async () => {
```

Comment:
Maybe im reading wrong but title says declare higher fee and code is with lower fee
initialFee - initialFee / 20

Reply:
Fixed, thanks!